redisvl/cli/utils.py

```diff
@@ -18,20 +18,15 @@ def create_redis_url(args: Namespace) -> str:
         url += args.host + ":" + str(args.port)
         return url
 
+
 def add_index_parsing_options(parser: ArgumentParser) -> ArgumentParser:
-    parser.add_argument(
-        "-i", "--index", help="Index name", type=str, required=False
-    )
+    parser.add_argument("-i", "--index", help="Index name", type=str, required=False)
     parser.add_argument(
         "-s", "--schema", help="Path to schema file", type=str, required=False
     )
     parser.add_argument("--host", help="Redis host", type=str, default="localhost")
     parser.add_argument("-p", "--port", help="Redis port", type=int, default=6379)
-    parser.add_argument(
-        "--user", help="Redis username", type=str, default="default"
-    )
+    parser.add_argument("--user", help="Redis username", type=str, default="default")
     parser.add_argument("--ssl", help="Use SSL", action="store_true")
-    parser.add_argument(
-        "-a", "--password", help="Redis password", type=str, default=""
-    )
+    parser.add_argument("-a", "--password", help="Redis password", type=str, default="")
```

Comment:
we will eventually need to add TLS support here in the form of a path to the server CA, and client/user certs (public and private) too. With the bedrock launch coming, it would be great to push users here for an easier onboarding :)